volatility3/cli/__init__.py

```diff
@@ -247,7 +250,12 @@ def run(self):
             default=[],
             action="append",
         )
-
+        parser.add_argument(
```

Comment:
Sorry, that's not a helpful comment.  It's just a feeling I get but it doesn't feel like it's been smoothly integrated into how everything works, but rather has lots of external moving bits (like files the user has to pass in, which get stashed in a non-unique place in the config to get used where they're needed.  As I say, I think there may be a better way that evades both the file issues, improves on the uniqueness and wouldn't require user interaction.

Reply:
The use of this cache might silently fill the filesystem without noticing the user, although the generated file is `xz` compressed. Indeed, everything could be stored in the cache, but then this would imply that `--clear-cache` command might also remove it, when trying to fix a symbol cache problem.

Reply:
The unintentionally filling the disk is a potential issue, but we can build cache checks on runs to clear out old files if it becomes a problem.  What's the expected filesize for one of these xz files once it's made?

We'd want `--clear-cache` to do exactly that, it's supposed to set you back to square 1, so you can rule the cache out as a contributing factor to a problem.